packages/common/src/types/job.ts

```diff
@@ -0,0 +1,9 @@
+export type Job = {
+    jobUuid: string;
+    projectUuid: string;
+    createdAt: Date;
+    updatedAt: Date;
+    jobStatus: string;
+    jobResults?: string;
```

Comment:
We can have a `ComplieJobResults` type:

```typescript
export type Job = {
  ...
  jobResults?: CompileJobResults,
  ...
}

export type CompileJobResults = {
  compiledProjectUuid: string
}
```

Reply:
Did you have another idea of what might go in `jobResults`? If we don't need it now we could also remove it